packages/@react-spectrum/card/test/CardView.test.js

```diff
@@ -1164,4 +1164,33 @@ describe('CardView', function () {
       expect(row.parentNode.style.height).toBe(`${mockHeight}px`);
     });
   });
+
+  // TODO: not testing waterfall layout because of aforementioned issue with the heights for each card being set to 0 for that layout
```

Comment:
Should this comment be changed to not say `todo`? Or a followup issue created?

Reply:
I can create a follow up issue for this